.github/workflows/release.yaml

```diff
@@ -6,16 +6,25 @@ on:
       - "v*"
 jobs:
   release:
+    permissions:
+      contents: write
+      id-token: write
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
           submodules: true
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@main
+        with:
+          cosign-release: 'v1.13.1'
+      - name: Install crane
+        uses: imjasonh/setup-crane@v0.1
       - name: authenticate
         run: |
           gh auth login --with-token <<<'${{ secrets.GITHUB_TOKEN }}'
           docker login docker.io --username ${{ secrets.DOCKERHUB_USER }} --password-stdin <<<'${{ secrets.DOCKERHUB_TOKEN }}'
       - name: release
         run: |
-          make -f builder/Makefile.release PUSH_DOCKER_REPO=aquasec/tracee
+          make -f builder/Makefile.release PUSH_DOCKER_REPO=developerguy/tracee
```

Comment:
I guess you're testing, right ? Could you move this to draft if you're only testing ? Thanks!

Reply:
we are ready, https://github.com/developer-guy/tracee/actions/runs/4026850058/jobs/6921861268

Reply:
we are ready https://github.com/developer-guy/tracee/actions/runs/4027024574/jobs/6922256638

Reply:
```shell
$ for i in latest 999.999.999 full full-999.999.999
  do
    COSIGN_EXPERIMENTAL=1 cosign verify developerguy/tracee:$i
  done
```